src/WebJobs.Script/Config/ExtensionRequirementOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.ExtensionRequirements;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    public class ExtensionRequirementOptions
```

Comment:
Does this have to be public?

Reply:
need it to be public for options to populate.